### DISCUSSION TITLE
Getting the edges for a clique

### DISCUSSION THREAD

**Discussion by vappiah** [networkx/networkx#5413] [Q&A] [answered]
Dear Developers,

I am interested in finding the list of edges in identified cliques. I have been able to use the find_cliques method to find the cliques. But my challenge is identifying the edges. I would be grateful if you can advice on what to do. 
Thanks.

**Comment by dschult** [comment, marked as answer]
Since it is a clique, the set of all edges is every possible edge between those nodes.  If you just want 2-tuples of the nodes involved you can use `itertools.combinations(nodes, 2)` or `permutations` depending on whether you want both directions or not. 
But you can also use `nx.complete_graph(nodes)` or `nx.complete_graph(nodes, create_using=nx.DiGraph)`. 

If you want the original edge attributes from the original graph, use those 2-tuple edges to look up the attributes on the original graph. with something like `G.edges[u,v]["attr"]` or `[(u, v, G[u][v]) for u,v in nx.complete_graph(nodes).edges]`

**Reply by vappiah** [reply]
Thanks. Your suggestions worked well.

By the way, I detected two other unusual behaviours from networkx. I will create another thread and report there.
